hazelcast/src/main/java/com/hazelcast/internal/nearcache/NearCacheRecordStore.java

```diff
@@ -110,7 +110,7 @@
      *                            size check, otherwise set {@code false} to evict
      *                            after a max size check.
      */
-    void doEviction(boolean withoutMaxSizeCheck);
+    boolean doEviction(boolean withoutMaxSizeCheck);
```

Comment:
minor: internal API but adding a `@return` would be nice

Reply:
fixed